SampleAPIApp/ViewController/MainTableViewController.m

```diff
@@ -575,9 +599,102 @@ - (MainTableViewCell *)tableView: (UITableView *)tableView
                 }
             }
         }
-        
+        else if ([[[NADeviceList gDeviceList] typeForModule:deviceId] isEqual:self.anemoModuleType])
+        {
+            if (indexPath.row == MEASURE_INDEX_ANEMO_WIND_ANGLE)
+            {
+                cell.title.text = self.titleAnemoWindAngle;
+                if ([[DataRetriever valueForType:NAMeasureTypeWindAngle inDictionary:data] isKindOfClass:[NSNumber class]] && dataReliableForModule)
+                {
+                    float angle = [[DataRetriever valueForType:NAMeasureTypeWindAngle inDictionary:data] floatValue];
+                    NSString *direction = [NSString new];
```

Comment:
To-fix : The part where you inferred the direction from the angle should be removed. For the time being, we want to keep this demo app only displaying raw data from our API. Once this is done, I will gladly accept you merge pull request. Thank you for your great work !